celery_slack/utils.py

```diff
@@ -0,0 +1,12 @@
+def ignore(*exceptions):
```

Comment:
I think we should name this something like `slack_ignore` to be more explicit. Let's also move it to a file called `decorators.py` rather than utils.